manual/mingw/tools/chocolateyInstall.ps1

```diff
@@ -50,10 +50,10 @@ if ($exception -eq 'sjlj') {
   $exceptionHandling64 = 'seh'
 }
 
-$url = "http://downloads.sourceforge.net/project/mingw-w64/Toolchains%20targetting%20Win32/Personal%20Builds/mingw-builds/$packageVersion/threads-$threads/$exceptionHandling/i686-$packageVersion-release-$threads-$exceptionHandling-rt_v3-$rev.7z"
-#http://downloads.sourceforge.net/project/mingw-w64/Toolchains%20targetting%20Win32/Personal%20Builds/mingw-builds/4.8.3/threads-posix/sjlj/i686-4.8.3-release-posix-sjlj-rt_v3-rev0.7z
-$url64 = "http://downloads.sourceforge.net/project/mingw-w64/Toolchains%20targetting%20Win64/Personal%20Builds/mingw-builds/$packageVersion/threads-$threads/$exceptionHandling64/x86_64-$packageVersion-release-$threads-$exceptionHandling64-rt_v3-$rev.7z"
-#http://downloads.sourceforge.net/project/mingw-w64/Toolchains%20targetting%20Win64/Personal%20Builds/mingw-builds/4.8.3/threads-posix/sjlj/x86_64-4.8.3-release-posix-sjlj-rev0.7z
+$url = "http://downloads.sourceforge.net/mingw-w64/i686-$packageVersion-release-$threads-$exceptionHandling-rt_v3-$rev.7z"
+#http://downloads.sourceforge.net/mingw-w64/i686-4.8.3-release-posix-sjlj-rt_v3-rev0.7z
+$url64 = "http://downloads.sourceforge.net/mingw-w64/x86_64-$packageVersion-release-$threads-$exceptionHandling64-rt_v3-$rev.7z"
+#http://downloads.sourceforge.net/mingw-w64/x86_64-4.8.3-release-posix-sjlj-rev0.7z

```

Comment:
That's it. I can not bother, but I think projects can re-arrange their
directory structure but the short link will still work.

On Wed, Jan 7, 2015 at 12:20 PM, Rob Reynolds <notifications@github.com>
wrote:

> In manual/mingw/tools/chocolateyInstall.ps1
> <https://github.com/ferventcoder/chocolatey-packages/pull/50#discussion-diff-22612743>
> :
>
> > @@ -50,10 +50,10 @@ if ($exception -eq 'sjlj') {
> >    $exceptionHandling64 = 'seh'
> >  }
> >
> > -$url = "http://downloads.sourceforge.net/project/mingw-w64/Toolchains%20targetting%20Win32/Personal%20Builds/mingw-builds/$packageVersion/threads-$threads/$exceptionHandling/i686-$packageVersion-release-$threads-$exceptionHandling-rt_v3-$rev.7z"
> > -#http://downloads.sourceforge.net/project/mingw-w64/Toolchains%20targetting%20Win32/Personal%20Builds/mingw-builds/4.8.3/threads-posix/sjlj/i686-4.8.3-release-posix-sjlj-rt_v3-rev0.7z
> > -$url64 = "http://downloads.sourceforge.net/project/mingw-w64/Toolchains%20targetting%20Win64/Personal%20Builds/mingw-builds/$packageVersion/threads-$threads/$exceptionHandling64/x86_64-$packageVersion-release-$threads-$exceptionHandling64-rt_v3-$rev.7z"
> > -#http://downloads.sourceforge.net/project/mingw-w64/Toolchains%20targetting%20Win64/Personal%20Builds/mingw-builds/4.8.3/threads-posix/sjlj/x86_64-4.8.3-release-posix-sjlj-rev0.7z
> > +$url = "http://downloads.sourceforge.net/mingw-w64/i686-$packageVersion-release-$threads-$exceptionHandling-rt_v3-$rev.7z"
> > +#http://downloads.sourceforge.net/mingw-w64/i686-4.8.3-release-posix-sjlj-rt_v3-rev0.7z
> > +$url64 = "http://downloads.sourceforge.net/mingw-w64/x86_64-$packageVersion-release-$threads-$exceptionHandling64-rt_v3-$rev.7z"
> > +#http://downloads.sourceforge.net/mingw-w64/x86_64-4.8.3-release-posix-sjlj-rev0.7z
>
> I mean besides one is shorter and has no spaces.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ferventcoder/chocolatey-packages/pull/50/files#r22612743>
> .
>